js/components/controlBar.js

```diff
@@ -858,7 +859,9 @@ var ControlBar = React.createClass({
               ariaLabel={CONSTANTS.ARIA_LABELS.MULTI_AUDIO}
               ariaHasPopup={true}
               ariaExpanded={this.props.controller.state.multiAudioOptions.showPopover ? true : null}
-              onClick={this.handleMultiAudioClick}
+              onClick={function(evt) {
```

Comment:
Is this change necessary? From the looks of the old code, it looks like the event argument should be passed to handleMultiAudioClick.

Reply:
Oh, yes! You are right! Thank you.